doc/whats_new.rst

```diff
@@ -22,6 +22,8 @@ Current
 Changelog
 ~~~~~~~~~
 
+- Added automatic conversion of FIF to BrainVision format with warning for EEG only data and conversion to FIF for meg non-FIF data, by `Alex Rockhill` _ (`#237 <https://github.com/mne-tools/mne-bids/pull/237>`_)
```

Comment:
```suggestion
- Added automatic conversion to BrainVision format for EEG only data and to FIF for meg (non-FIF) data, by `Alex Rockhill` _ (`#237 <https://github.com/mne-tools/mne-bids/pull/237>`_)
```

Reply:
Actually, this will not render correctly because you don't have your name down below with link to website

Reply:
Fixed